pkg/apis/kops/networking.go

```diff
@@ -124,6 +124,9 @@ type RomanaNetworkingSpec struct {
 
 // AmazonVPCNetworkingSpec declares that we want Amazon VPC CNI networking
 type AmazonVPCNetworkingSpec struct {
+	// The container image name to use, which by default is:
+	// 602401143452.dkr.ecr.us-west-2.amazonaws.com/amazon-k8s-cni:1.0.0
+	ImageName string `json:"imageName,omitempty"`
```

Comment:
Nit: We probably shouldn't specify the default explicitly - rather we just allow overriding of the image name with this field.  Technically defaults are part of the API, and we're not guaranteeing this value as the default forever :-)